src/main/java/io/vertx/core/http/impl/ClientConnection.java

```diff
@@ -80,8 +80,8 @@
   // Requests can be pipelined so we need a queue to keep track of requests
   private final Queue<HttpClientRequestImpl> requests = new ArrayDeque<>();
   private final Handler<Throwable> exceptionHandler;
-  private final HttpClientMetrics metrics;
   private final Object metric;
+  final HttpClientMetrics metrics;
```

Comment:
no problems with that. (I don't mind package protected members when they are final)